utils/procps-ng/Makefile

```diff
@@ -73,7 +73,7 @@ define GenPlugin
    $(call Package/procps-ng/Default)
    DEPENDS:=procps-ng
    TITLE:=Applet $(2) from the procps-ng package
-   DEFAULT:=y
+   DEFAULT:=n
```

Comment:
Why is this change being made?

Reply:
Because with the default, when procps-ng was selected it automatically selected every sub-package procps has, with the default as `n` only the needed package is selected.

Reply:
That doesn't seem right. I have a feeling it has to do with the dependency order. See https://github.com/openwrt/packages/blob/master/net/seafile-server/Makefile#L45 as an example.

It does not select all of it.

Reply:
To make it clearer, +procps-ng makes +procps-ng-ps available.

Reply:
I just tested it. It seems I'm wrong. Can you open a separate pull request for the procps-ng issue?

Reply:
Yes sir! should I redo all patches ?

Reply:
Sure. A single commit is easiest.

Reply:
But do I need to remove the change to procps-ng from the kea's commit?

Reply:
Yes.

Reply:
It's done @neheb